### PR TITLE
Move all Trivy Scans to one workflow

### PR DESCRIPTION
**Pull request by udosson**
This PR combines all Trivy scans of the Indy-Node images in one workflow using the matrix strategy.
Also, this PR added the scan of the recent bullseye image contributed by @mgmgwi. 
Further, the status badge of the build workflow was updated because the status badges weren't working any since #40 

A successful run of the workflow can be found [here](https://github.com/udosson/docker-container-wg/actions/runs/1266155403). The event trigger was changed temporarily.

### CODE REVIEW

File: .github/workflows/trivy-all.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      matrix:
+        os_version: [ bullseye, buster, ubuntu16, ubuntu18]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Run Trivy on indy_node:${{ matrix.os_version }}
+        uses: aquasecurity/trivy-action@master
+        with:
+          image-ref: 'ghcr.io/idunion/indy-node-container/indy_node:main-${{ matrix.os_version }}'
```
Comment (mgmgwi) [inline .github/workflows/trivy-all.yml:24]:
```suggestion
          image-ref: 'ghcr.io/idunion/indy-node-container/indy_node:latest-${{ matrix.os_version }}'
```
I think we should check the latest released version here.

**Review by mgmgwi — CHANGES_REQUESTED** [review]
We should check the latest released version - IMO.

File: .github/workflows/trivy-all.yml (hunk 1 above)
Comment (udosson) [reply .github/workflows/trivy-all.yml]: done.

**Review by mgmgwi — APPROVED** [review]
LGTM

**Pull request merged**